assets/src/stories-editor/components/right-click-menu/index.js

```diff
@@ -100,16 +208,53 @@ const RightClickMenu = ( props ) => {
 				{
 					name: __( 'Duplicate Block', 'amp' ),
 					blockAction: duplicateBlock,
+					params: [ firstBlockClientId ],
 					icon: 'admin-page',
 					className: 'right-click-duplicate',
 				},
 			);
 		}
 
+		const pageList = getBlockOrder();
+		const numPages = pageList.length;
+		if ( block && numPages > 1 ) {
+			const currentPage = getCurrentPage();
+			const currentPagePosition = pageList.indexOf( currentPage );
+			if ( currentPagePosition > 0 ) {
+				const prevPage = getPageByOffset( -1 );
+				if ( isBlockAllowedOnPage( block.name, prevPage ) ) {
+					blockActions.push(
+						{
+							name: __( 'Send block to previous page', 'amp' ),
+							blockAction: moveBlockToPage,
+							params: [ prevPage ],
+							icon: 'arrow-left-alt',
```

Comment:
I'd use `select( 'core/block-editor' ).getSettings().isRTL` and switch the arrow icons when a RTL language is being used

Reply:
So we support RTL ATM?

Reply:
For the most part. RTL stylesheets are created automatically, which helps with the majority of things. There's currently an issue with the layout (see #3012), but that's no reason to neglect it.

Reply:
RTL support - https://github.com/ampproject/amp-wp/pull/3417/commits/3527c8ad3c7bedc073ee1c37e43888704e294d46